app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientView.kt

```diff
@@ -180,6 +183,14 @@ class LinkIdWithPatientView(
         .build()
   }
 
+  override fun renderPatientName(patientName: String) {
+    idPatientNameTextView.text = Truss()
+        .append(resources.getString(R.string.linkidwithpatient_patient_text))
+        .append(patientName)
```

Comment:
I see from the designs that the patient name has a different text appearance than the rest of the text, but we don't seem to be using the different style here for it.

Reply:
Never mind, just noticed that the layout has two different text views now.

Reply:
However, in this case, using `Truss` is unnecessary since it is meant to be used in situations where we want to dynamically style different sections of text.